sci-libs/HighFive/HighFive-2.8.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DOCS_BUILDER="doxygen"
+DOCS_DIR="doc"
+
+inherit cmake docs
+
+DESCRIPTION="Header-only C++ interface for libhdf5"
+HOMEPAGE="https://github.com/BlueBrain/HighFive"
+SRC_URI="https://github.com/BlueBrain/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="Boost-1.0"
+SLOT="0"
+IUSE="mpi test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	sci-libs/hdf5[mpi?]
+	doc? ( app-doc/doxygen[dot] )
```

Comment:
This is probably not correct, I doubt we need doxygen at runtime.  Also you can use `DOCS_DEPEND` to specify this dependency.

Reply:
Great thanks for review. I completely forgot about `DOCS_DEPEND` feature.
The `app-text/doxygen[dot]` dependency is moved to this variable now.